static/sw.js

```diff
@@ -0,0 +1,13 @@
+importScripts('/bower_components/sw-toolbox/sw-toolbox.js');
+
+toolbox.options.debug = true;
+toolbox.router.default = toolbox.fastest;
+
+const filesToCache = [
+    '/',
+    'https://fonts.googleapis.com/css?family=Roboto:regular,bold,italic,thin,light,bolditalic,black,medium&lang=en'
+];
+
+self.addEventListener('install', function(e) {
```

Comment:
toolbox.precache doesn't actually go in an install handler, just in the body of the script. Toolbox creates it's own install handler, and if they get executed in the wrong order you might find that precaching isn't happening.